jeffjxu/Wesley-Central-Access-master
# list inquiries in edit client

list inquires for a client when editing that client
how to test:
- go to an edit client page
- see that the client's inquires are listed (check that the client actually has inquires)
